packages/block-editor/src/components/page-template-picker/picker.native.js

```diff
@@ -36,13 +46,16 @@ const __experimentalPageTemplatePicker = ( {
 
 	const [ templatePreview, setTemplatePreview ] = useState();
 	const [ pickerVisible, setPickerVisible ] = useState( visible );
+	const [ tooltip, setTooltipVisible ] = useState( false );
```

Comment:
NP: maybe this should be `const [ tooltipVisible, setTooltipVisible ]`, wdyt?

Reply:
Oh yes! Thanks for the suggestion, I'll change it =)

Reply:
+1 to @mkevins comment here.

Reply:
Updated https://github.com/WordPress/gutenberg/pull/21974/commits/a00bbb6274664ca63bbb4a09a581494e08d1e342